kubernetes/template.yaml.jinja

```diff
@@ -78,17 +79,45 @@ spec:
         - "--job_name={{ job }}"
         - "--worker_hosts={{ worker_hosts() }}"
         - "--ps_hosts={{ ps_hosts() }}"
+        volumeMounts:
+{% if credential_secret_name != "" %}
+        - name: credential
+          mountPath: /etc/credential
+          readOnly: true
 {% endif %}
+{% if job == "worker" and i == 0 %}
+        - name: traindir
```

Comment:
As I know, result of training is located in the traindir of worker-0. So I provide a volume https://github.com/tensorflow/ecosystem/pull/22/files#diff-12dba5811d3565440c4c55f399a1b116R118 for worker-0 container and tensorboard container. The tensorboard container also have a volume mount as https://github.com/tensorflow/ecosystem/pull/22/files#diff-12dba5811d3565440c4c55f399a1b116R110 show.